canopeum_frontend/src/utils/validators.ts

```diff
@@ -1,4 +1,4 @@
-const passwordRegex = /^(?=.*[A-Za-z])(?=.*\d)[\d!#$%&*?@A-Za-z]{8,}$/u
+export const passwordRegex = /^(?=.*[A-Za-z])(?=.*\d)[\d!#$%&*.?@A-Za-z]{8,}$/u
```

Comment:
Just noticed this is also missing `_`

This restriction feels very arbitrary. I'm pretty sure our backend and DB can take any unicode character

This regex also groups "numbers" with "special characters"

how about something like this instead?

```suggestion
// At least 1 alpha, 1 numerical. Length of at least 8.
export const passwordRegex = /^(?=.*[A-Za-z])(?=.*\d)(?=.{8,}).*$/u
```

And if you wanna force special characters:
```suggestion
// At least 1 alpha, 1 numerical, 1 special. Length of at least 8.
export const passwordRegex = /^(?=.*[A-Za-z])(?=.*\d)(?=.*[\W_])(?=.{8,}).*$/u
```
(ref: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Regular_expressions/Character_class_escape#w and https://www.w3schools.com/jsref/jsref_regexp_wordchar_non.asp )

Reply:
For now we don't ask for special characters in the password validators so I'll leave it to the first option